types/backbone/index.esm.d.ts

```diff
@@ -0,0 +1,595 @@
+/// <reference types="jquery" />
+/// <reference types="underscore" />
+
```

Comment:
You cannot just remove the `export = Backbone` and `export as namespace Backbone` without replacing it by an equivalent default export. `Backbone` is an object that does not just contain all other objects but that also mixes in the `Events` interface and of which the `sync` property is supposed to be writable.